.github/workflows/workflow.yaml

```diff
@@ -123,7 +123,7 @@ jobs:
     runs-on: "ubuntu-latest"
     if: startsWith(github.ref, 'refs/tags/v') && github.repository == 'open-policy-agent/gatekeeper'
     needs: [lint, build_test, helm_build_test]
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
should we keep this one the same to be consistent with the previous job? Also this is a bigger jump